packages/core/src/components/datetime-picker/datetime-picker.scss

```diff
@@ -12,17 +12,14 @@
 
 :host {
   @include ix-component;
+  --btn-width: 100%;
```

Comment:
What is the idea of using custom css property here? This should not be a setting which a user should customize.

Use should use a scss variable if it used internal.